docs/manage/config-policies.rst

```diff
@@ -56,6 +56,12 @@ or CLI.
    both the invariant config and constraints for a given scope are updated together. It is not
    possible to update only one of these components independently.
 
+.. note::
+    
+   Admins are encouraged to draft and save their config policies for each scope in separate YAML or 
+   JSON files. This allows for consistency between config policies set in the WebUI and CLI and
+   lessens the chance for unintended changes to be made to a given scope's policies.
+
```

Comment:
Order is good. 

I don't agree with the reasons for the suggestion though. The config policies will be consistent if set via webUI or CLI whether admins draft them separately or not. That makes it sound like there's a bug in our APIs or something.

I think the reason to suggest admins draft it separately is that our webUI does not support common helpful features like "save", "undo", or any kind of history.

Reply:
>  The config policies will be consistent if set via webUI or CLI whether admins draft them separately or not. That makes it sound like there's a bug in our APIs or something.

Hmm, I guess I meant consistency on their end (so that they don't have one config used for the webUI policies and a different folder of configs whose paths are input into the CLI). 
but that makes sense! I'll rephrase the benefits of the note